### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by Art4**
In this PR I will prepare everything for release SimplePie 1.7.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,12 +5,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/simplepie/simplepie/compare/1.6.0...master)
+## [Unreleased](https://github.com/simplepie/simplepie/compare/1.7.0...master)
+
+## [1.7.0](https://github.com/simplepie/simplepie/compare/1.6.0...1.7.0) - 2022-08-??
```
Comment (Art4) [inline CHANGELOG.md:10]: The date still needs to be adjusted.

**Comment by Art4** [comment]
If no further changes or merges are planned, the date can be adjusted and this PR can be merged. After that we are ready for release 1.7.0.

**Comment by Art4** [comment]
@mblaney What do you think about releasing 1.7.0 soon? I would like to continue working on refactoring and finalizing more features that are planned for the 1.8 release.

**Comment by mblaney** [comment]
sure, sorry I keep holding you up @Art4. I've just merged a few small patches, if you want to pick a date next week for the changelog I will make sure the release happens.

**Comment by Art4** [comment]
@mblaney No problem, I will update the PR for the changes of #744 and #745 and update the date. Do you have a suggestion for a specific date?

**Comment by mblaney** [comment]
yeah let's go for 30th of September

**Comment by Art4** [comment]
Done. Everything is ready for takeoff.

**Pull request merged**